### DISCUSSION TITLE
Can a quad_form constraint be DPP?

### DISCUSSION THREAD

**Discussion by sarahxdean** [cvxpy/cvxpy#2569] [Q&A] [answered]
If `x` is a variable and `P` is a parameter (declared to be PSD), is it possible for their `quad_form` to be DPP?

In particular I'm interested in a constraint of the following form 
```quad_form(x, P) <= q```

**Comment by Transurgeon** [comment, marked as answer]
Hello @sarahxdean , unfortunately what you want is not directly supported in cvxpy.
However, there are definitely ways around it. I invite you to read the following [section](https://www.cvxpy.org/tutorial/dpp/index.html#the-dpp-ruleset) in the documentation (which includes an answer to your question, using sum_of_squares instead of quad_form).
Best of luck!

**Reply by sarahxdean** [reply]
This is perfect, thank you!
